src/modules/shared/pages/innovation/sections/section-info.component.html

```diff
@@ -2,172 +2,128 @@
 
   <div class="nhsuk-grid-row">
     <div class="nhsuk-grid-column-full">
-
-      <div class="nhsuk-u-margin-top-0">
-        <span class="nhsuk-u-visually-hidden">Information: </span>
-
-        <p class="d-flex">
-
-          <ng-container [ngSwitch]="section.status.id">
-
-            <ng-container *ngSwitchCase="'NOT_STARTED'">
-              <theme-svg-icon type="error"></theme-svg-icon>
-              <span class="nhsuk-u-margin-right-2"></span>
-              <span>This section has not been started yet.</span>
-            </ng-container>
-
-            <ng-container *ngSwitchCase="'DRAFT'">
-              <theme-svg-icon type="edit"></theme-svg-icon>
-              <span class="nhsuk-u-margin-right-2"></span>
-              <span *ngIf="isInnovatorType; else notInnovator">This section is in draft and it is not visible to others. Confirm answers and complete section to become visible again.</span>
-              <ng-template #notInnovator><span>This section is in draft.</span></ng-template>
-            </ng-container>
-
-            <ng-container *ngSwitchCase="'SUBMITTED'">
-              <ng-container *ngIf="section.date !== '' && section.submittedBy !== null">
-                <theme-svg-icon type="success"></theme-svg-icon>
+    
+      <ng-container>
+        <div class="nhsuk-u-margin-top-0">
+          <span class="nhsuk-u-visually-hidden">Information: </span>
+          <p class="d-flex">
+            <ng-container [ngSwitch]="sectionInfo.status.id">
+              
+              <ng-container *ngSwitchCase="'NOT_STARTED'">
+                <theme-svg-icon type="error"></theme-svg-icon>
                 <span class="nhsuk-u-margin-right-2"></span>
-                <span>
-                  This section was submitted by {{ section.submittedBy.name }} on {{ section.date | date: ('app.date_formats.long_date' | translate) }}.
```

Comment:
On figma we have a different phrase: "Last submitted by Clarice Johnson (Collaborator) on 24 October 2021"